.github/workflows/publish_charm.yaml

```diff
@@ -63,6 +63,53 @@ jobs:
           docker pull $image
           imageId=$(docker images $image --format "{{.ID}}")
           charmcraft upload-resource $charmName ${{ matrix.image }}-image --image=$imageId --verbosity=trace
+  lookup-external-images:
+    name: Lookup External Images
+    runs-on: ubuntu:latest
+    outputs:
+      images: ${{ env.IMAGES }}
+    steps:
+      - uses: actions/checkout@v3
+      - if: inputs.working-directory != './'
+        name: Change directory
+        run: |
+          TEMP_DIR=$(mktemp -d)
+          cp -rp ${{inputs.working-directory}}/. $TEMP_DIR
+          rm -rf .* * || :
```

Comment:
that's like `rm -rf .* * || true`, right?

Reply:
Yes, `:` is like a no-op in shell.